### PR TITLE
Fix tiling issues for PLANTLIKE and FIRELIKE with FSAA enabled

### PR DESCRIPTION
**Pull request by RealBadAngel**

### CODE REVIEW

File: src/script/common/c_content.cpp [hunk 1]
```diff
@@ -258,17 +258,20 @@ void push_object_properties(lua_State *L, ObjectProperties *prop)
...
 /******************************************************************************/
-TileDef read_tiledef(lua_State *L, int index)
+TileDef read_tiledef(lua_State *L, int index, u8 drawtype)
 {
 	if(index < 0)
 		index = lua_gettop(L) + 1 + index;
 
 	TileDef tiledef;
-
+	bool default_tiling = (drawtype == NDT_PLANTLIKE || drawtype == NDT_FIRELIKE)
```
Comment (est31) [inline src/script/common/c_content.cpp]: Perhaps use boolean operations instead?

**Comment by est31** [comment]
Very nice, will have closer look later.

File: src/script/common/c_content.cpp (hunk 1 above)
Comment (est31) [inline src/script/common/c_content.cpp]: Perhaps its even better in this case to have explicit values here, boolean operations are hackier here.

**Comment by est31** [comment]
Looks good, but others should have a look at it too, before merging.

**Comment by RealBadAngel** [comment]
https://github.com/minetest/minetest/commit/3295f3c40157c2bc385a903611bfa803a69b617d

**Pull request closed without merging**